src/client/sandbox/event/focus-blur.js

```diff
@@ -140,7 +140,7 @@ export default class FocusBlurSandbox extends SandboxBase {
                     this.eventSimulator[bubblesEventType](el, relatedTarget);
                     this.timersSandbox.deferFunction(() => this.eventSimulator[type](el, relatedTarget));
                 }
-                else if (browserUtils.isMSEdge && type === 'blur') {
+                else if (browserUtils.isMSEdge && browserUtils.version < 17 && type === 'blur') {
```

Comment:
As we will not support the previous MSEdge version (16) and `17` version will update soon on appropriate Windows systems, we can assume that `MSEdge` has only `17` and remove all condition with `browserUtils.version` here and below.